src/peergos/server/tests/MerkleBtree.java

```diff
@@ -209,7 +223,7 @@ public void storageSize() throws Exception {
         }
 
         int size = ((RAMStorage)tree.storage).size();
-        if (size != 30)
+        if (size != 31)
```

Comment:
``31`` is a nice prime, perhaps add a comment about why it is the expected value of ``size``?

Reply:
I added 1 because of the signing key now being stored in the dht.